### DISCUSSION TITLE
How to conduct LES uncertainty analysis and How the very large fluid/wall exchanges affect the relevance of the simulations?

### DISCUSSION THREAD

**Discussion by Remedy-J** [firemodels/fds#13646] [Q&A]
Hello, experts and scholars

First of all I want to thank you for looking at my question, thank you very much.

I used FDS to construct the scene of the underground parking lot and carried out the fire simulation. I ran simulations with different wind speeds, and the fire simulation involved multiple cars.My FDS revision is FDS-6.8.0-0-g886e009-release. I submitted my simulation results to a journal, the reviewers asked me two questions:

1. If the wind is involved, how is it modelled? What is the turbulent intensity, and how were turbulent fluctuations modelled?  I assume LES was used in this study. LES uncertainty analysis is required for an LES simulation.

2.  Wall flows are very significant due to the very large surface area presented by the vehicles; can you indicate this developed surface area? To what extent can the very large fluid/wall exchanges affect the relevance of the simulations? Are there other studies in the literature with FDS that have raised this point? How can these limitations affect the conclusions?

I have two doubts about the questions raised by the reviewers. First, I didn't make any changes to the MISC, leaving it completely default. Is the airflow modeled using VLES in this case? How should I conduct LES uncertainty analysis? I looked up other articles on fire simulation using FDS and found no reference to LES uncertainty analysis.

Second, as the reviewer said: "To what extent can the very large fluid/wall exchanges affect the relevance of the simulations? How can these limitations affect the conclusions?”  I checked the literature on FDS fire simulation and found no other articles discussing this issue.

I still can't understand the reviewer's meaning clearly. How do I conduct LES uncertainty analysis? How should I revise the manuscript and answer the reviewer's questions?

Thank you again for taking the time to read my question and thank you for your help.

**Comment by mcgratta** [comment]
The FDS Technical Reference Guide should provide answers to your questions. If there is a specific detail that is not clear, post the question here, but please be specific. You basically asking how FDS handles (1) turbulence, and (2) boundary layers. These are very wide topic areas. That is why we write these manuals.

As for defaults, you, the modeler, are responsible for all assumptions you make. Leaving something as the default is an assumption.
